src/VirtualClient/VirtualClient.Actions.UnitTests/Blender/BlenderBenchmarkMetricsParserTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using NUnit.Framework;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Reflection;
+    using VirtualClient.Contracts;
+
+    [TestFixture]
+    [Category("Unit")]
+    public class BlenderBenchmarkMetricsParserTests
+    {
+        private string ExamplePath
+        {
+            get
+            {
+                string workingDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+                return Path.Combine(workingDirectory, "Examples", "Blender");
+            }
+        }
+
+        private IList<Metric> actualMetrics { get; set; }
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            string outputPath = Path.Combine(ExamplePath, "results_example.json");
+            string rawText = File.ReadAllText(outputPath);
+
+            BlenderBenchmarkMetricsParser testParser = new BlenderBenchmarkMetricsParser(rawText);
+            this.actualMetrics = testParser.Parse();
+        }
+
+
+        [Test]
+        public void BlenderMetricsParserTestsCorrectly_Monster()
```

Comment:
You might want to think about parsing multiple output files in the same Tests.cs. You could potentially setup each test separately. Up to you

Reply:
sounds good. Will open another PR with test scores from different devices (CPU/AMD GPU/NVDA GPU)